### ISSUE TITLE
Sending 'stop' via the Commands Tab in the GTalkSMS UI after a 'ring' command was send results in "unkown command" error on notification address.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#309]
```
Hi, would it be possible to add a feature to stop the ringing on the phone 
side? when typing the stop command inside gtalksms's commands tab, I get

'stop': unknown command. Send "?" for help.

Sent to me via google talk.

It would be nice to fix that, but besides that what I really want is some easy 
to access button to stop it from ringing, like when it's ringing, it puts up a 
message box with a button you can press to stop it from ringing.

Thanks! :)
```

Original issue reported on code.google.com by `azelp...@azelphur.com` on 30 Dec 2012 at 7:11
- Blocking: #311

**Comment by GoogleCodeExporter** [comment]
```
We are talking about ringing caused by the ring command and not because someone 
is calling you, correct?

'stop' should always work if the ring command is enabled. This sounds like a 
bug. Could you please provide some debug log when this happens? 
http://code.google.com/p/gtalksms/wiki/GenerateAppLog

```

Original comment by `fschm...@gmail.com` on 31 Dec 2012 at 1:10
- Changed title: **Allow the user to stop the ringing from phone after issuing the 'ring' command.**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Correct, I have sent the debug log as requested :)

Steps to reproduce:
Run ring command
Open GtalkSMS and go to the commands section
In the box at the bottom, type "stop"

What should happen:
Ringing should stop

What actually happens:
Error message: 'stop': unknown command. Send "?" for help. is sent to my Google 
Talk account
```

Original comment by `azelp...@azelphur.com` on 1 Jan 2013 at 5:08

**Comment by GoogleCodeExporter** [comment]
```
Thank you very much. The comment #2 makes the problem more clear to me and I 
was able to reproduce this.
```

Original comment by `fschm...@gmail.com` on 2 Jan 2013 at 12:42
- Changed title: **Sending 'stop' via the Commands Tab in the GTalkSMS UI after a 'ring' command was send results in "unkown command" error on notification address.**

**Comment by GoogleCodeExporter** [comment]
```
And note that sending 'stop' from the notification address does in fact stop 
the ringing (just as it was intended).
```

Original comment by `fschm...@gmail.com` on 2 Jan 2013 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 4 Jan 2013 at 12:37
- Added labels: **Type-Defect**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 4 Jan 2013 at 12:38
- Now blocking: #311

**Comment by GoogleCodeExporter** [comment]
```
"stop" command is not a real one.
You can use "ring:stop" to do the job.

The issue is that the "stop" command is not managed by the intent COMMAND.
```

Original comment by `Florent....@gmail.com` on 27 Jan 2013 at 1:42

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 6730eb6707e8.
```

Original comment by `yako...@gmail.com` on 27 Jan 2013 at 11:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 30 Jan 2013 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
New version 4.4 Beta 1 is available in the download section
```

Original comment by `Florent....@gmail.com` on 16 Feb 2013 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4
```

Original comment by `Florent....@gmail.com` on 15 Sep 2013 at 3:27
- Changed state: **Deployed**
